client/src/api/results.js

```diff
@@ -0,0 +1,79 @@
+export async function fetchResults() {
+  // TODO: Connect API here later
+  /*
+  const response = await fetch("localhost:3001/app/offers", {
+    header: {
+      //TODO:
+    },
+  });*/
+
+  //const result = await response.json();
+
+  const mockedResults = [
+    {
+      id: 1,
+      title: "5 Bananen",
+      category: "fruits",
+      location: "50968 Köln",
+      pickup_date: "15.08.2020",
+      pickup_time: "15:00 - 16:00",
```

Comment:
Any reason you don't use camelCase here?

If you save dates and times like this, you will face issues comparing and sorting these values. Instead, I would suggest using a unix timestamp or `Date`.

Reply:
Was not happy yet with those names- prefer these changes aswell.

Reply:
Will change this in later steps.